docs/demos/podman-ai-lab-to-rhoai/podman-ai-lab-to-rhoai.md

```diff
@@ -0,0 +1,447 @@
+        ```
+        oc apply -k ./components/openshift-servicemesh/operator/overlays/stable
+        ```
+    2. Run the following oc command to deploy Serverless.
+        ```
+        oc apply -k ./components/openshift-serverless/operator/overlays/stable
+        ```
+
+    3. Wait until the Service Mesh and Serverless operators have installed successfully. 
+        ```
+        watch oc get csv -n openshift-operators
+        ```
+
+        ![Operator Status](img/csv_status.png)
+
+    4. We'll be using the single stack serving in OpenShift AI so we'll want use a trusted certificate instead of a self signed one. This will allow our chatbot to access the model inference endpoint.
+
+          Run the below oc commands
+
+          1. Get the name of the ingress cert we will need to copy. Select a secret that has **cert** in the name.
+              ```
+              oc get secrets -n openshift-ingress | grep cert
+              ```
+
+              ![Ingress Cert Secret Name](img/ingress_cert_secret.png)
+
+          2. Copy the full name of the secret and replace the name in the below oc command. Make sure you're in the top level directory of this project and run the below command.
+              ```
+              oc extract secret/ingress-certs-2024-06-03 -n openshift-ingress --to=ingress-certs --confirm
+              ```
+
+              You should now have a *ingress-certs* directory with a tls.crt and tls.key file.
+
+              ![Ingress Cert Directory](img/ingress_certs_directory.png)
+
+          3. We'll now update the secret that will be used in our OpenShift AI data science cluster.
+
+              ```
+              cd ingress-certs
+
+              oc create secret generic knative-serving-cert -n istio-system --from-file=. --dry-run=client -o yaml | oc apply -f -
+
+              cd ..
+              ```
+
+              **NOTE:** *You can delete the ingress-certs folder after you have created the knative-serving-cert secret.*
+
+    5. Run the following oc commands to enable the Single Model Serving runtime for OpenShift AI. 
+        ```
+        oc apply -k ./components/model-server/components-serving
+        ```
+
+    6. It will take around 5 to 10 minutes for the changes to be applied. Single-model serving should be ready when Service Mesh and Serverless have the below instances created. Open the OpenShift web console and go to **Operators -> Installed Operators**.
+
+        ![Knative Instance](img/knative_instance.png)
+
+        ![Service Mesh Instance](img/servicemesh_instance.png)
+
+2. Go to the OpenShift AI dashboard and expand **Settings** and select **Serving Runtimes**. You should now see that Single-model serving enabled at the top of the page.
+
+    **NOTE:** *You might need to refresh the page and it could take a few minutes for the changes to be applied.*
+
+    ![Single serving enabled](img/rhoai_single_serving_enabled.png)
+
+### Add a Custom Serving Runtime
```

Comment:
Add a note that we are using a GGUF model just for the sake of the demo.
The whole purpose of RHOAI and the transition from Podman AI to it is to be able to "scale" and use real models (vs quantized) on a supported environment. If we only reuse the same quantized model here there is not much point in using RHOAI.

Reply:
Added "We will continue to use the GGUF version of the model to be able to deploy this model without the need for a hardware accelerator (e.g. GPU). OpenShift AI contains a scalable model serving platform to accommodate deploying multiple full sized LLMs."